README.md

```diff
@@ -24,6 +24,10 @@ What it's going to do:
 
 People sometimes run into problems with database connection issues. If that's the case, chances are postgres is shutting down right after starting up and to find out why, run `brew info postgresql` and check out the instructions on how to start postgresql manually. That's usually enough to get people unblocked.
 
+If you'd like some seed data to exercise the site run:
+
+```$ bin/rails db:seed```
```

Comment:
ℹ️ script/dev-setup is supposed to be idempotent, in that you can run it 5 or 6 times in a row and it shouldn't mess anything up. But we had it setup to run `rails db:setup` which meant that it was blowing away our local data and re-running the seeds every time you ran it, so I pulled that out of there and made adding the seed data a separate step.

Reply:
Feedback: Maybe we should add a `start` script that runs dev-setup and then `bin/rails server`? That way it's one command to update dependencies and get the site up.

Reply:
Ok! I took care of a couple pieces of your feedback all at once:

* In `script/dev-setup`, I added a new wrapper I created called `db:seed_if_necessary` that only runs if the database is empty so new users get seed data and everyone else gets to keep their local customizations.
* Added a `script/dev-start` script that runs `script/dev-setup` before `bin/rails server`. I don't think I'll run it myself though, I feel like the server takes a long time to start up already.